source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -77,19 +77,48 @@ content: |
              .build();
         
            ClientEncryption clientEncryption = ClientEncryptions.create(clientEncryptionSettings);
+           BsonBinary dataKeyId = clientEncryption.createDataKey(kmsProvider, new DataKeyOptions());
 
   .. note::
 
      Ensure that the client has `ReadWrite
      <https://docs.mongodb.com/manual/reference/built-in-roles/#readWrite>`_
      permissions on the specified key vault namespace.
 ---
-title: Verify the data encryption key was created
+title: Verify that the Data Encryption Key was Created
 ref: verify-data-key-created
 level: 4
 content: |
-  Verify that the key document was created in your MongoDB replica set in the
-  key vault collection. This document contains the following data:
+  First, verify that the command to create the data encryption key returns the
+  ID of the newly-created key:
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        The ``createDataKey()`` method returns a :java-docs-latest:`BsonBinary
+        <javadoc/org/bson/BsonBinary.html>` object from which we can extract
+        the UUID and Base64 representations of the key ID.
```

Comment:
I think we should let the user know here why this matters.

UUID is how the shell presents data key IDs and represents the container of the data.

the base64 representation is required for constructing the extended JSON Binary Data representation, and is required for automatic json schema.

Reply:
Good idea. I will add information on when each representation is used.